main.cpp

```diff
@@ -98,6 +98,31 @@ struct Computer
         bool setPrice(float newPrice = 100.0f);
         //output cuda version and number of cores
         std::string outputCUDAVersionAndCores();//returns a string representing the cuda capbility 
+
+        void boostTheGraphics()
+        {
+            int testValue = 0;
+            while(isGSyncCapable)
+            {
+                testValue = 1;
+                std::cout << "Boosted!" << std::endl;
+                break;
+            }
```

Comment:
This loop runs exactly once.
there is nothing in this that actually changes `isGSyncCapable`, which would break it out of the loop
the only thing that exits the loop is the `break` statement and that is executed the first time the loop is entered.
- [ ] revise

Reply:
made the loop based on a decrementing value